  
syntax = "proto3";

// input image, width, height
message B64Image {
    string b64image_content = 1;
    string b64image_style = 2;
	    
}
// output img

message out_img {
    string b64image = 3;
}


// service
service StylizeImage{
    rpc Stylized_Image(B64Image) returns (out_img) {}
}